src/controllers/userController.js

```diff
@@ -3,30 +3,54 @@ const { catchAsync } = require("../middlewares");
 
 const getUserSignUp = catchAsync(async (req, res, next) => {
   const {
-    body: { userId, nickname, password, age, gender },
+    body: { userId, nickname, password, age, gender, admin },
     file,
   } = req;
 
-  let profileImg = (file) ? file.path : "NULL";
+  let profileImg = file ? file.location : NULL;
 
   if (!userId || !nickname || !password || !age) {
     const error = new Error("Please write your Info");
     error.statusCode = 400;
     throw error;
   }
-  
   await userService.getUserSignUp(
     userId,
     nickname,
     gender,
     password,
     profileImg,
-    age
+    age,
+    admin
   );
 
   res.status(201).json({ message: `Welcome ${nickname}!` });
 });
 
+const signIn = catchAsync(async (req, res, next) => {
+    const { userId, password } = req.body;
+    if (!userId || !password) {
+    const err = new Error("CONFIRM INPUT ID OR PASSWORD");
+    err.statusCode = 400;
+    throw err;
+  }
+  const result = await userService.signIn(userId, password);
+  return res.status(200).json(result);
+});
```

Comment:
가독성을 고려해서 코드를 작성해주세요.
```suggestion
const signIn = catchAsync(async (req, res, next) => {
    const { userId, password } = req.body;
    
    if (!userId || !password) {
        const err = new Error("CONFIRM INPUT ID OR PASSWORD");
        err.statusCode = 400;
        throw err;
    }
    
  const result = await userService.signIn(userId, password);
  
  return res.status(200).json(result);
});
```